app/pages/events/event_edit.php

```diff
@@ -26,10 +26,10 @@
 $start_date = $v->date("start_date")->label("Date de départ")->required();
 $end_date = $v->date("end_date")
     ->label("Date de retour")->required()
-    ->after($start_date->value, "Doit être après le départ");
+    ->min($start_date->value, false, "Doit être après le départ");
 $limit_date = $v->date("limit_date")
```

Comment:
Nice job not adding attributes for min and max here, this is what we want I think

Reply:
Thx bru